tasks/Task.cpp

```diff
@@ -93,13 +93,17 @@ bool Task::configureHook()
     cVb.buildFrom(cMb);
     task.set_cVe(cVb);
 
+    //set the time for trigger the timeout
+    timeout = base::Timeout(base::Time::fromSeconds(1));
```

Comment:
I added it in order to send a vector of zeros if the timeout is elapsed. 
Without this, the velocity controller goes to timeout so the robot tends to float. In that case, the expected behavior is keep the vehicle in the position if no markers are detected.